aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -77,50 +77,56 @@ This interface is implemented by a concrete type, `MyDependency`:
 
 [!code-csharp[](dependency-injection/samples/3.x/DependencyInjectionSample/Services/MyDependency.cs?name=snippet1)]
 
-`MyDependency` requests an <xref:Microsoft.Extensions.Logging.ILogger`1> in its constructor. It's not unusual to use dependency injection in a chained fashion. Each requested dependency in turn requests its own dependencies. The container resolves the dependencies in the graph and returns the fully resolved service. The collective set of dependencies that must be resolved is typically referred to as a *dependency tree*, *dependency graph*, or *object graph*.
+In the sample app, the `IMyDependency` service is registered with the concrete type `MyDependency`. The registration service lifetime is <xref:Microsoft.Extensions.DependencyInjection.ServiceCollectionServiceExtensions.AddScoped*>, the lifetime of a single request. [Service lifetimes](#service-lifetimes) are described later in this topic.
```

Comment:
@Rick-Anderson Curious as to why this is AddScoped?
Since the dependency was explicitly created in the example above, feels it is more to be a Transient (Just from the context of this example). I might be missing something here.

Reply:
We could change it to transient but I don't see much advantage. The sample is used to explain how to use dependency injection. It's not a good sample because as we show later, the logger is already in the DI container. I'd like to have a simple sample to demonstrate DI  where the code required DI.

@serpent5  should we change the sample to transient?